decisions/0015-env-triad-terms.md

```diff
@@ -0,0 +1,52 @@
+---
+status: proposed / in progress
+date: 2024-08-28
+deciders: Montana Smith, Mark Miller, Sierra Moxen
+consulted: Lee Ann McCue, Natalie Winans
+informed: 
+---
+# Establishing valid terms for MIxS Environmental Triad 
+
+## Context and Problem Statement
+
+Determining what to assert for each of the environmental triad slot (`env_broad_scale`, `env_local_scale`, and `env_medium`) is difficult.
+NMDC will provide a reporducible logic to provide users with a curated lists of valid terms for each slot.
+
+These terms will not be limiting, a user can still use string match to enter terms. This is a temporart allowance until user research and refinment of these queries is completed.
```

Comment:
The original had two independent clauses separated by a comma. I changed it to a period.

This suggestion incorporates @aclum's feedback also.

```suggestion
These terms will not be limiting. A user can still use string match to enter terms. This is a temporary allowance until user research and refinment of these queries is completed.
```

I don't know what the author meant by "string match" (e.g. entry of freeform strings)?